trailing_controller_node:
  ros__parameters:
    max_lateral_distance: 3.0
    max_longitudinal_distance: 20.0
    prefer_ahead: true
    
    desired_gap: 2.5
    min_gap: 1.5
    max_gap: 4.0
    
    velocity_kp: 0.8
    velocity_kd: 0.3
    velocity_ki: 0.05
    
    max_velocity: 7.0
    min_velocity: 0.5
    emergency_brake_distance: 1.0
    
    wheelbase: 0.33
    max_steer: 0.42
    
    stanley_k: 1.2
    stanley_ks: 0.5
    
    pp_lookahead_gain: 0.6
    pp_lookahead_min: 1.0
    pp_lookahead_max: 3.5
    
    switch_velocity: 2.5
    control_rate: 20.0
    
    prediction_time: 0.5
    use_prediction: true
